src/module.ts

```diff
@@ -59,7 +59,13 @@ export default defineNuxtModule<ModuleOptions>({
     const isOriginSet = typeof usedOrigin !== 'undefined'
     if (!isOriginSet) {
       // TODO: see if we can figure out localhost + port dynamically from the nuxt instance
-      usedOrigin = 'http://localhost:3000'
```

Comment:
lets:
- remove the todo as we finished
- not make the protocol dynamic -> this is always http in development